### ISSUE TITLE
resnet18 and 36

### ISSUE THREAD

**Issue by lolongcovas** [KaimingHe/deep-residual-networks#37]
I am trying to train resnet 18 and 36 from scratch using imagenet12 train dataset. The preprocessing is the same as the [Facebook Torch implementation](https://github.com/facebook/fb.resnet.torch/blob/master/datasets/transforms.lua), however, the top 1 accuracy on the validation set is still very low (49%), quite below the accuracy reported by the paper. I wonder if these resnet18 and 36 will be released as well.

Best

**Comment by nnop** [comment]
It's ResNet-34 commonly called.
